packages/loader/container-loader/src/container.ts

```diff
@@ -1743,7 +1732,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
 
         // The relative loader will proxy requests to '/' to the loader itself assuming no non-cache flags
         // are set. Global requests will still go directly to the loader
-        const loader = new RelativeLoader(this.loader, () => this.containerUrl);
+        const loader = new RelativeLoader(this.loader);
```

Comment:
can the constructor just take the container? why do we need to late inject it with resolveContainer below?